src/components/menus/TertiaryNav/examples/TertiaryNavWithDrawerExample.tsx

```diff
@@ -0,0 +1,90 @@
+/* @ts-nocheck */
```

Comment:
Nice! I love the inclusion of "recipe examples"

Does StoryBook expose these or are they just standalone for visual reference only?